src/js/Controllers/TTSController.js

```diff
@@ -18,6 +88,20 @@ class TTSController {
             case "SetGlobalProperties":
                 return true
             case "Speak":
+                var ttsChunks = rpc.params.ttsChunks
+                this.filePlaylist = []
+                for (var i=0; i<ttsChunks.length; i++) {
+                        this.filePlaylist.push(ttsChunks[i])
+                }
+
+                if(this.filePlaylist.length > 0) {
+                    if(this.filePlaylist[0].type === "FILE") {
+                        this.playAudio();
+                    } else if (this.filePlaylist[0].type === "TEXT"){
+                        this.speak();
+                    }
+                }
+                
```

Comment:
Should also be added to these (assuming they are implemented):
1. `Alert` 
2. `PerformAudioPassThru`
3. `AlertManeuver`

Reply:
These are covered by TTS.SPEAK on the HMI side